crates/utils/src/lib.rs

```diff
@@ -14,3 +15,12 @@ pub fn convert_allocation_operator_set_to_rewards_operator_set(
         id: operator_set.id,
     }
 }
+
+pub fn convert_stake_registry_strategy_params_to_registry_coordinator_strategy_params(
```

Comment:
I feel like we should create our own `StrategyParams` to use on all interfaces, with `From` and `Into` implementations for each of the equivalent types.

Reply:
will create an issue for it

Reply:
and similar utils

Reply:
Yes, sure! Let's leave it for another PR